src/main/groovy/io/gatling/gradle/GatlingRunTask.groovy

```diff
@@ -84,7 +84,11 @@ class GatlingRunTask extends DefaultTask {
         }
     }
 
-    Iterable<String> simulationFilesToFQN() {
-        return SimulationFilesUtils.resolveSimulations(project, this.simulations)
+    List<String> simulationClasses() {
+        def classpath = project.configurations.gatlingRuntimeClasspath.getFiles()
+        def includes = this.simulationClass? List.of(simulationClass) : project.gatling.includes
+        def excludes = this.simulationClass? List.of() : project.gatling.excludes
```

Comment:
nit: format
```suggestion
        def includes = this.simulationClass ? List.of(simulationClass) : project.gatling.includes
        def excludes = this.simulationClass ? List.of() : project.gatling.excludes
```

(this is the ternary operator, not the nullable operator of kotlin)

Reply:
Done